kubernetes/container-azm-ms-agentconfig.yaml

```diff
@@ -191,6 +191,11 @@ data:
     # [agent_settings.proxy_config]
     #    ignore_proxy_settings = "true"  # if this is not applied, default value is false
 
+    # Uses fluent-bit for perf and container inventory instead of fluentd
+    # Disables custom metrics
```

Comment:
isnt that custom metrics continue to work?

Reply:
this is for present. in future yes this will no longer disable custom metrics.